ReactiveCocoa/Swift/Event.swift

```diff
@@ -49,7 +57,7 @@ public enum Event<T, E: ErrorType> {
 	public func map<U>(f: T -> U) -> Event<U, E> {
 		switch self {
 		case let .Next(value):
-			return Event<U, E>.Next(Box(f(value.unbox)))
+			return Event<U, E>.Next(Box(f(value.value)))
```

Comment:
Yeah, I had some thoughts about it. The changes are mainly for users who are using/would like to use this library for iOS 7 target (I am one of them) by importing the sources to an app target directly.

In the case, `Result`'s `Box` is visible in the entire app target (because that is included in the target), including RAC sources. To avoid conflict between `Result`'s `Box` and RAC's `Box`, the user must comment out either of them. If `box.value` and `box.unbox` are both exists, either A or B becomes uncompilable with the commenting out (so the user must fix the getters of which one of the two).

This is why I unified the unboxing getter.